### PR TITLE
Fix iOS rn-fetch-blob warnings

### PR DESCRIPTION
**Pull request by cristi-lupu**
Jira: [CXIOS-6232](https://jira.tenkasu.net/browse/CXIOS-6232)

### What has been done?
1. I've moved the implementation of **`NSString (Contains)`** in a separate **`.m`** file.

### How to test?
Test this PR https://github.com/crunchyroll/cr-ios-app/pull/2954

### Checklist

- [x] I've run tests before opening PR

[Reviewer Guidelines](https://github.com/crunchyroll/ios-resources/blob/master/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: ios/RNFetchBlob.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -136,6 +139,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```
Comment (batanus) [inline ios/RNFetchBlob.xcodeproj/project.pbxproj:142]: I hope this auto-generated Xcode change will not break something

**Review by batanus — APPROVED** [review]
LGTM

File: ios/RNFetchBlob.xcodeproj/project.pbxproj (hunk 1 above)
Comment (cristi-lupu) [reply ios/RNFetchBlob.xcodeproj/project.pbxproj:142]: No, will not. CocoaPods copies only the files specified in podspec and there are only source code files.

**Review by gugennota — APPROVED** [review]

**Comment by gugennota** [comment]
@cristi-lupu Please update pod reference from the main PR

**Pull request merged**